setup.py

```diff
@@ -17,8 +17,9 @@
     packages=find_packages('src'),
     install_requires=['requests', 'six', 'virtualenv-api', 'pbxproj', 'pyinstaller'],
     extras_require={
-        ':platform_machine=="x86_64"': 'wxPython',
-        ':sys_platform=="darwin"': 'dmgbuild',
+        ':sys_platform=="darwin"': ['dmgbuild'],
+        ':sys_platform=="darwin" and platform_machine=="x86_64"': ['wxPython'],
```

Comment:
Windows is the only platform that currently requires wxPython, as Mac (and now GTK!) have alternate options, so maybe we should switch this to Windows? One piece I need to think through more is how best to streamline the backend setup. On platforms like Linux, you could setup Android, GTK, or wx backends (or multiple), so probably best to let the user pick and not try to install all of those from setup.py.

For now I guess we can do it through docs, but it'd be nice if there was somehow a way to script or automate the setup of a backend.